app/articles/page.tsx

```diff
@@ -1,29 +1,31 @@
 import Link from "next/link";
 import { notFound } from "next/navigation";
-
+import styles from "./index.module.scss";
 import { httpGet$GetPosts } from "@/modules/commands/GetPosts/fetcher";
+import Navbar from "../_containers/Navbar";
+import Footer from "../_containers/Footer";
+import Posts from "./_containers/SectionPosts";
 import { intentionallyIgnoreError } from "@/modules/error/intentionallyIgnoreError";
 
+
 export default async function Page() {
-  const data = await httpGet$GetPosts(`/wp-json/wp/v2/posts`, {})
+  const data = await httpGet$GetPosts(`/wp-json/wp/v2/posts`, {categories: 3})  // 3 coresponding to "article" category
```

Comment:
create constant call ARTICLE_CATEGORY_ID in this file.

we may have to export it to other files in this app.